jxline-core/src/test/java/ProtocolTest.java

```diff
@@ -0,0 +1,48 @@
+import cloud.xline.jxline.Client;
+import cloud.xline.jxline.ProtocolClient;
+import com.google.protobuf.ByteString;
+import com.xline.protobuf.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import static org.assertj.core.api.Assertions.*;
+
+@Timeout(value = 20)
+public class ProtocolTest {
+    static ProtocolClient client;
+
+    @BeforeAll
+    static void onConnect() {
+        client = Client.builder().endpoints("http://172.20.0.5:2379").build().getProtocolClient();
```

Comment:
The address is hard coded, is it portable?

Reply:
Added a static variable.

Reply:
I didn't get your point, which variable?

Reply:
`static String INIT_ENDPOINT = "http://172.20.0.5:2379";`

Reply:
So where is it?

Reply:
https://github.com/xline-kv/java-xline/pull/10/commits/e68fbbae97dc8f93800dbcd0a0bca120583a3877